test_files/es6/enum.js

```diff
@@ -9,7 +9,6 @@ EnumTest1.XYZ = 0;
 /** @type {EnumTest1} */
 EnumTest1.PI = 3.14159;
 /** @typedef {number} */
-// This additional exported enum is here to exercise the fix for issue #51.
```

Comment:
I thought the type was ignored, but now that I'm making a smaller test case I see that I might be wrong!  I will revisit this once I confirm it.